recipes/azure-common/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.1.3" %}
+
+package:
+  name: azure-common
+  version: {{ version }}
+
+source:
+  fn: azure-common-{{ version }}.zip
+  url: https://pypi.python.org/packages/48/89/017df9bed55925de004076937ca168d3a03aa0250fbda7826ec3c0ac714f/azure-common-{{ version }}.zip
+  md5: 13471213f49400c5d483865c96558ab9
+  
+build:
+  script: pip install --no-deps .
```

Comment:
> I took the pip install from the discussion, where it seemed people were
happy with it. But I've switched it to python setup.py install instead

That discussion never settled on a new standard and we ended up using the
"old way," but you are correct. We considered pip and we may be using it
again eventually.